docs/source/conventions.md

```diff
@@ -73,6 +73,41 @@ Mixing singular and plural is confusing.
 For example, using `/users` for retrieving users, and `/user/21` to retrieve a single user.
 
 
+### URL Attributes (singular vs plural)
+
+**Rule:** Use plural for attributes that can contain one or multiple items. Use singular for attributes that can contain only one single item.
```

Comment:
I'm not sure what you mean by URL attributes, which is not a term I've seen used before. #1295 was talking about querystring parameters but the examples here look like path segments. Maybe you want this to apply to both?

Reply:
Do you want to rename a whole bunch of endpoints and deprecate the other form?  If not we end up with a non consistent API.

Reply:
I am referring to attributes that we pass via GET requests though the URL.

Reply:
I'm pretty sure these are usually called query string params or parameters, not attributes. If I search for url attribute on google it actually corrects it and shows results for url parameters

Reply:
ah, got it, yes, that should be called parameters.

Reply:
Changed it. Thx for the feedback.